public/electron.js

```diff
@@ -1,16 +1,16 @@
 // TODO #151: Can't use ES7 import statements here?
 
 // Modules to control application life and create native browser window
-const { app, BrowserWindow, dialog } = require("electron");
 const path = require("path");
+const { app, BrowserWindow, dialog } = require("electron");
 const ipc = require("electron").ipcMain;
```

Comment:
Why is this separated into two different lines? Could it be written like this?

```suggestion
const { app, BrowserWindow, dialog, ipcMain: ipc } = require("electron");
```

Reply:
That worked! Thanks!